### DISCUSSION TITLE
QPage overflow hidden not working

### DISCUSSION THREAD

**Discussion by JokerMartini** [quasarframework/quasar#11032] [General - Components / Directives / etc] [answered]
How can I make the index.vue page have overflow hidden but remain filling the full screen. Similar to what you see on google maps? Where the map extends past the viewport but it doesn't have a scroll bar.

Currently it creates the svg but adds scroll bars which i don't want. I'd prefer there not to be scrollbars since it should be overflow-hidden, but that doesn't seem to work either. 

![image](https://user-images.githubusercontent.com/1416749/137803153-5b6244ae-643e-409d-8595-3b2f880e0fbc.png)

Here is a working codepen
[https://codepen.io/jokermartini/pen/GRvZLNV](https://codepen.io/jokermartini/pen/GRvZLNV)

**MainLayout.vue**
```
<template>
  <q-layout view="hHh lpr lFf">
    <q-header elevated>
      <q-toolbar class="bg-primary">
        <q-btn flat dense round icon="mdi-menu" @click="toggleLeftDrawer" />

        <q-toolbar-title> Application</q-toolbar-title>
      </q-toolbar>
    </q-header>

    <q-drawer
      v-model="leftDrawerOpen"
      show-if-above
      bordered
      behavior="desktop"
    >
      ...
    </q-drawer>

    <q-page-container>
      <router-view />
    </q-page-container>
  </q-layout>
</template>
```

**index.vue**
```
<template>
  <q-page>
      <div style="background: yellow">
      <svg viewBox="0 0 100 100">
        <ellipse cx="50" cy="80" rx="46" ry="19" fill="#07c" />
        <path
          d="M43,0c-6,25,16,22,1,52c11,3,19,0,19-22c38,18,16,63-12,64c-25,2-55-39-8-94"
          fill="#e34"
        />
        <path d="M34,41c-6,39,29,32,33,7c39,42-69,63-33-7" fill="#fc2" />
      </svg>

      <q-page-sticky position="bottom-right" :offset="[18, 18]">
        <q-btn fab @click.stop="" color="primary" icon="mdi-refresh">
          <q-tooltip>Reset Zoom</q-tooltip>
        </q-btn>
      </q-page-sticky>
    </div>
  </q-page>
</template>

```

**Comment by hawkeye64** [comment]
Please create a CodePen

**Reply by JokerMartini** [reply]
I tried to create a Codesandbox but that wouldn't let me create one with my github account which is more ideal since it is setup more like my project with all the source folders and whatnot. I can try CodePen for the time being.

**Reply by JokerMartini** [reply]
I've created a codepen showing the issue https://codepen.io/jokermartini/pen/GRvZLNV

**Comment by pdanpdan** [comment, marked as answer]
https://codepen.io/pdanpdan/pen/vYJGMJw?editors=1010

**Reply by JokerMartini** [reply]
thank you
